vasyl_varyshchuk/Week_1/task_3/generate_file_size.py

```diff
@@ -2,46 +2,49 @@
 This module returns random size of file
 """
 
+import os
 import random
 
 list_of_files = ['reports.txt', 'monthly_reports.zip', 'README.txt',
                  'picture.jpg', 'logo.ttf', 'config.ini']
 
 
-def recognize_file_type(a_file):
-    return a_file.split('.')[1]
+def recognize_file_type(file_name):
+    _, extension = os.path.splitext(file_name)
```

Comment:
Well, I showed you an example from google just to see that guys are using unpacking for extraction of file extension.
Here you don't need to import additional modules.

it should work as I mentioned previously:

     filename, extension = file_name.split('.')